ci/build-test.sh

```diff
@@ -1,20 +1,29 @@
 #!/usr/bin/env bash
 
+### This script should only be run on Improbable's internal build machines.
+### If you don't work at Improbable, this may be interesting as a guide to what software versions we use for our
+### automation, but not much more than that.
+
 set -e -u -o pipefail
 
 if [[ -n "${DEBUG-}" ]]; then
   set -x
 fi
 
+if [[ -z "$BUILDKITE" ]]; then
+  echo "This script is only to be run on Improbable CI."
+  exit 1
+fi
+
 cd "$(dirname "$0")/../"
 
-source scripts/pinned-tools.sh
+source ci/common.sh
 
-echo "--- Build Tools.sln :construction:"
-dotnet build tools/Tools.sln
+setupPremerge
 
-echo "--- Test DocsLinter.csproj :link:"
-dotnet test --logger:"nunit;LogFilePath=$(pwd)/logs/docslinter-test-results.xml" "tools/DocsLinter/DocsLinter.csproj"
+mkdir -p ./logs
 
-echo "--- Test ReleaseTool.csproj :fork:"
-dotnet test --logger:"nunit;LogFilePath=$(pwd)/logs/releasetool-test-results.xml" "tools/ReleaseTool.Tests/ReleaseTool.Tests.csproj"
+echo "--- Running premerge :running:"
+docker run \
```

Comment:
This should probable have `--rm`
https://docs.docker.com/engine/reference/run/#clean-up---rm